src/Stack/Config.php

```diff
@@ -37,6 +37,16 @@ public static function loadDefaults()
         self::defineFromEnv("MEMCACHED_HOST", "");
         self::defineFromEnv("MEMCACHED_DISCOVERY_HOST", "");
 
+        $page_cache_key_prefix = env("STACK_PAGE_CACHE_KEY_PREFIX") ?: "";
+        if (env("STACK_PAGE_CACHE_BACKEND") == "redis") {
+            self::defineHostnamePortFromEnv("RT_WP_NGINX_HELPER_REDIS", "", "STACK_PAGE_CACHE_REDIS_HOST_PATH");
```

Comment:
Just as a small suggestion: we can use connection URI's details
```
memcached://<host>:<port>
redis://user:password@host:port/
```
and then just parse the protocol to find out which backend we're using.

Reply:
We've decided it makes more sense to have separate vars for host, port and eventually password. Password is currently not supported in memc_pass afaik, also not available in redis_pass, but there are workarounds for that, also not implemented in nginx-helper afaik. We can add a URI variable later if we find out it's needed.